crates/ruff/src/rules/flake8_pyi/rules/import_from_future.rs

```diff
@@ -0,0 +1,33 @@
+use rustpython_parser::ast::StmtImportFrom;
+
+use ruff_diagnostics::{Diagnostic, Violation};
+use ruff_macros::{derive_message_formats, violation};
+
+use crate::checkers::ast::Checker;
+
+#[violation]
+pub struct ImportFromFuture;
```

Comment:
I think i'd call this rule `FutureAnnotationsInStub` which would fit the [other namein PYI](https://beta.ruff.rs/docs/rules/#flake8-pyi-pyi) better